tests/e2e/e2e_workflow.spec.js

```diff
@@ -18,11 +18,10 @@ describe('Basic end-to-end Workflow', function () {
   it('Installs angular-cli correctly', function() {
     this.timeout(300000);
 
+    sh.exec('npm link', { silent: true });
```

Comment:
[npm link](https://docs.npmjs.com/cli/link) doesn't seem to take a path to link globally, only to link locally. Is there a missing argument in these docs that you know of?